myst_nbs/case_studies/stochastic_volatility.myst.md

```diff
@@ -6,13 +6,20 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.7
 kernelspec:
-  display_name: Python 3 (ipykernel)
+  display_name: pymc_env
   language: python
-  name: python3
+  name: pymc_env
 ---
 
+(notebook_name)=
```

Comment:
Do not use `notebook_name` literally: https://docs.pymc.io/en/latest/contributing/jupyter_style.html#first-cell

Reply:
Oof, that was a big oversight.